README.md

```diff
@@ -1 +1,39 @@
-# Awesome-book
\ No newline at end of file
+# Awesome-book
+
+> This is the first JS Module Project we are working on. We will make a webpage to store books, add and remove them from the list and much more. 
+
+
+![screenshot](/images/demo.png)
+
+Our challenge was to build a webpage that can store our favourite books using classes, arrays and so on. [Live Demo](). 😊
+
+### Features implemented:
+* Add and Remove functions
+* 
```

Comment:
- [ ] Let's keep our documentation neat. This item is blank and has no body, kindly remove it or add something to it.